frame/examples/offchain-worker-ping-pong/src/lib.rs

```diff
@@ -0,0 +1,681 @@
+			let results = signer.send_signed_transaction(|_account| {
+				// nonce is wrapped into a call to `pong_signed` public function
+				// of this pallet. This means that the transaction, when
+				// executed, will simply call that function passing `nonce` as
+				// an argument.
+				Call::pong_signed { nonce }
+			});
+
+			for (acc, res) in &results {
+				match res {
+					Ok(()) => log::info!("[{:?}] Submitted pong with nonce {}", acc.id, nonce),
+					Err(e) => log::error!("[{:?}] Failed to submit transaction: {:?}", acc.id, e),
+				}
+			}
+		}
+
+		Ok(())
+	}
+
+	/// A helper function to sign payload and send an unsigned pong transaction
+	fn ocw_pong_unsigned_for_any_account(
+		block_number: BlockNumberFor<T>,
+	) -> Result<(), &'static str> {
+		let pings = <Pings<T>>::get();
+		for p in pings {
+			let Ping(nonce) = p;
+
+			// -- Sign using any account
+			let (_, result) = Signer::<T, T::AuthorityId>::any_account()
+				.send_unsigned_transaction(
+					|account| PongPayload { nonce, block_number, public: account.public.clone() },
+					|payload, signature| Call::pong_unsigned_with_signed_payload {
+						pong_payload: payload,
+						signature,
+					},
+				)
+				.ok_or("No local accounts accounts available.")?;
+			result.map_err(|()| "Unable to submit transaction")?;
+		}
+
+		Ok(())
+	}
+
+	/// A helper function to sign payload and send an unsigned pong transaction
+	fn ocw_pong_unsigned_for_all_accounts(
+		block_number: BlockNumberFor<T>,
+	) -> Result<(), &'static str> {
+		let pings = <Pings<T>>::get();
+		for p in pings {
+			let Ping(nonce) = p;
+
+			// -- Sign using all accounts
+			let transaction_results = Signer::<T, T::AuthorityId>::all_accounts()
+				.send_unsigned_transaction(
+					|account| PongPayload { nonce, block_number, public: account.public.clone() },
+					|payload, signature| Call::pong_unsigned_with_signed_payload {
+						pong_payload: payload,
+						signature,
+					},
+				);
+			for (_account_id, result) in transaction_results.into_iter() {
+				if result.is_err() {
+					return Err("Unable to submit transaction")
+				}
+			}
+		}
+
+		Ok(())
+	}
+
+	/// A helper function to send a raw unsigned pong transaction.
+	fn ocw_pong_raw_unsigned(block_number: BlockNumberFor<T>) -> Result<(), &'static str> {
+		let pings = <Pings<T>>::get();
+		for p in pings {
+			let Ping(nonce) = p;
+			// nonce is wrapped into a call to `pong_unsigned` public function
+			// of this pallet. This means that the transaction, when executed,
+			// will simply call that function passing `nonce` as an argument.
+			let call = Call::pong_unsigned { block_number, nonce };
+
+			// Now let's create a transaction out of this call and submit it to
+			// the pool. Here we showcase two ways to send an unsigned
+			// transaction / unsigned payload (raw)
+			//
+			// By default unsigned transactions are disallowed, so we need to
+			// whitelist this case by writing `UnsignedValidator`. Note that
```

Comment:
What is `UnsignedValidator`?

Reply:
c2368f44f6f98aaa33dfa3c7db3f36e2159faac1